### ISSUE TITLE
No entryPoint defined for this router when using default tls config for entryPoint

### ISSUE THREAD

**Issue by LolHens** [containous/traefik#6442]
### Do you want to request a *feature* or report a *bug*?

Bug

#### static configuration:
```yml
entryPoints:
  web:
    address: ":80"
  websecure:
    address: ":443"
    http:      # these two lines
      tls: {}  # cause the error
```

#### dynamic configuration:
```yml
http:
  routers:
    test:
      entryPoints:
        - websecure
      rule: "Host(`test.example.com`)"
      service: test
```

If I comment out the last two lines in this static configuration snippet the error doesn't happen.
Note that this only seems to happen on the file provider since I have also specified entrypoints for routers via the docker swarmMode provider and it works just fine there.

### What did you expect to see?
The router was not using my configured entryPoint "websecure" like the one below in the screenshot but was assigned to "traefik" and "web".
The default tls configuration from #6417 seems to override my configured entrypoints.

### What did you see?
![image](https://user-images.githubusercontent.com/1524059/76097103-67a1a980-5fc7-11ea-9ed5-79c6c1ab62b9.png)

#### Log entry:
```
time="2020-03-06T15:17:54Z" level=debug msg="No entryPoint defined for this router, using the default one(s) instead: [traefik web websecure]" routerName=test
```

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
2.2.0-rc1
```

**Comment by ldez** [comment]
Thank you very much for testing the RC :heart:

duplicate of #6441

**Comment by traefiker** [comment]
Closed by #6444.
